MAINTAINERS.md

```diff
@@ -27,16 +29,25 @@ Review pull requests regularly, comment, suggest, reject, merge and close. Accep
 
 ### Triage Open Issues
 
-Review issues regularly and triage by labelling issues. For example, add "help wanted" to good issues for new community members. Request for more information from a submitter if an issue is not clear. Create new labels as needed by the project.
+Manage labels, review issues regularly, and triage by labelling them. For example, add "help wanted" to good issues for new community members and *blocker* for issues that scare you or need immediate attention. Request for more information from a submitter if an issue is not clear. Create new labels as needed by the project.
```

Comment:
You've pulled out some of the specifics of what our labels currently are. Considering we're iterating on them, that might be smart (I'd hate to roll this out to 25+ repos and decide we wanted to change from "minor" to "trivial" or something like that..  

What do you think of a putting out a separate doc with label information into .github?

Reply:
It's a good idea. I think I'd like to leave this text as is for now as it says "For example, .." but as we try to standardize labels we can extract.